tests/phpunit/Integration/JSONScript/TestCases/outline-02.json

```diff
@@ -87,6 +95,16 @@
 					"</ul>"
 				]
 			}
+		},
+		{
+			"type": "parser",
+			"about": "#2 (`outlineproperties=Assigned to` with a same page assigned to two people)",
+			"subject": "SRF/Outline/Q.3",
+			"assert-output": {
+				"not-contain": [
+					"<li class=\"mw-empty-elt\"></li>"
```

Comment:
Not the most safe assertion. This test could easily not be doing what is intended, or stop doing that, while still passing. 

Is it possible to instead check that the two values are both present?

Reply:
I added the check of the two values.